packages/jest-resolve/src/resolver.ts

```diff
@@ -485,16 +489,9 @@ export default class Resolver {
   }
 
   getMockModule(from: string, name: string): string | null {
-    const mock = this._moduleMap.getMockModule(name);
-    if (mock) {
-      return mock;
-    } else {
-      const moduleName = this.resolveStubModuleName(from, name);
-      if (moduleName) {
-        return this.getModule(moduleName) || moduleName;
-      }
-    }
-    return null;
+    const resolvedName = this.resolveStubModuleName(from, name) ?? name;
```

Comment:
Isn’t it that the current logic is optimised for better performance? I mean, resolving can be expansive step, so isn’t it better to skip it if the module is already cached in the `_moduleMap`?